src/App.jsx

```diff
@@ -1,42 +1,49 @@
-import './App.css'
-import Error from './pages/Error';
-import Notes from './pages/Notes';
-import Body from './pages/Body';
-import Navbar from './components/Navbar';
-import Footer from './components/Footer';
-import { createBrowserRouter,Outlet,Link } from "react-router-dom";
-
+import "./App.css";
+import Error from "./pages/Error";
+import Notes from "./pages/Notes";
+import Body from "./pages/Body";
+import Navbar from "./components/Navbar";
+import Footer from "./components/Footer";
+import Contact from "./components/Contact";
+import Testimonial from "./components/Testimonial";
+import { createBrowserRouter, Outlet } from "react-router-dom";
 
 function App() {
-
   return (
-    <>
-    <Navbar/>
-    <Outlet/>
-    <Footer/>
-    </>
-  )
+    <div className="flex flex-col justify-center gap-y-32">
+      <Navbar />
+      <Outlet />
+      <section id="testimonials">
+        <Testimonial />
+      </section>
+      <section id="contact">
+        <Contact />
+      </section>
+      <Footer />
+    </div>
+  );
 }
 
 export const appRouter = createBrowserRouter([
-{
-
-  path: "/",
-  element: <App/>,
-  errorElement: <Error/>,
-  children: [
-    {
-      path: "/",
-      element: <Body/>
-    },
-    {
-      path: "/notes",
-      element: <Notes/>
-    }
-  ]
-
-}
-
-])
-
-export default App
+  {
+    path: "/",
+    element: <App />,
+    errorElement: <Error />,
+    children: [
+      {
+        path: "/",
+        element: (
+          <section id="home">
```

Comment:
Yaha pe section kyu dala?

Reply:
react-scroll ni chalega bina sections ke, ya to components hi div ki jgah sections me wrap kr ke usme id set kr lo